internal/testdata/dlasqtest/commands.txt

```diff
@@ -0,0 +1,9 @@
+# Advice: nDiv in dlasq2 can be an effective counter for isolating a bug by using os.Exit(1) and "STOP" when nDiv hits a certain number.
+
+gfortran -o dlasq1test dlamch.f lsame.f xerbla.f ieeeck.f iparmq.f ilaenv.f dlasrt.f dlaisnan.f disnan.f dlasq6.f dlasq5.f dlasq4.f dlasq3.f dlasq2.f dcopy.f dlas2.f dlascl.f dlasq1.f testdlasq1.f90
```

Comment:
If possible, I would prefer having a `Makefile` instead, and if I may be really annoying, then also a `README.md` with just a short explanation what this is, what it does, what are the textfiles it generates.

This `Makefile` works for me, untested on OSX:
```make
FC = gfortran
FFLAGS = -O2

targets = testdlasq1 testdlasq2 testdlasq3 testdlasq4
objects = dlamch.o lsame.o xerbla.o ieeeck.o iparmq.o ilaenv.o dlasrt.o \
          dlaisnan.o disnan.o dlasq6.o dlasq5.o dlasq4.o dlasq3.o dlasq2.o

default : $(targets)
.PHONY : default

testdlasq1 : testdlasq1.o $(objects) dcopy.o dlas2.o dlascl.o dlasq1.o

testdlasq2 : testdlasq2.o $(objects)

testdlasq3 : testdlasq3.o $(objects)

testdlasq4 : testdlasq4.o $(objects)

% : %.o
    $(FC) $(FFLAGS) $^ -o $@

%.o : %.f90
    $(FC) $(FFLAGS) -c -o $@ $<

clean :
    rm -f *.o $(targets) *.txt
.PHONY : clean
```